### DISCUSSION TITLE
part-ii/2012-1/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#200] [Question comments]
Part II 2012 Algebraic Geometry: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2012-1/

**Comment by eha1234** [comment]
<details><summary>Spoilers</summary>

$\nu_p(\omega) = 0 $ except at $[0:1:0]$ where it is 4. $g=3$
</details>
